eng/pipelines/templates/steps/analyze.yml

```diff
@@ -0,0 +1,22 @@
+parameters:
+  - name: RustToolchain
+    type: string
+    default: stable
+
+steps:
+  - template: /eng/pipelines/templates/steps/use-rust.yml@self
+    parameters:
+      Toolchain: ${{ parameters.RustToolchain }}
+
+  - pwsh: |
+      . ./eng/common/scripts/Helpers/CommandInvocation-Helpers.ps1
+
+      $env:RUSTDOCFLAGS = "-D warnings"
+      $env:RUSTFLAGS = "-Dwarnings"
+
+      Invoke-LoggedCommand "cargo +$(RustToolchain) check -p azure_core --no-default-features"
+      Invoke-LoggedCommand "cargo +$(RustToolchain) install cargo-readme"
```

Comment:
This isn't even being used. Let's get rid of this before. If/when we add third-party dependencies, I want to re-review now that it's in official source.

Reply:
done